nexus/external-api/src/lib.rs

```diff
@@ -2662,6 +2662,39 @@ pub trait NexusExternalApi {
         path_params: Path<params::UpdatesGetRepositoryParams>,
     ) -> Result<HttpResponseOk<TufRepoGetResponse>, HttpError>;
 
+    /// Get the current target release of the rack's system software
+    ///
+    /// This may not correpond to the actual software running on the rack
+    /// at the time of request; it is instead the release that the rack
+    /// reconfigurator should be moving towards as a goal state. After some
+    /// number of planning and execution phases, the software running on the
+    /// rack should eventually correspond to the release described here.
+    #[endpoint {
+        method = GET,
+        path = "/v1/system/update/target-release",
```

Comment:
Thoughts on making this `/v1/system/target-release` and calling the operation `target_release_get` (or view)?

Reply:
Right, `80f690` renames it to `Unspecified`, and `87b20001e` makes the version mandatory, i.e., you can no longer set the target release to `Unspecified`.

Reply:
What about dropping the `update` in the URL path?  I'm not sure why that feels not quite right to me.  I don't feel strongly about it.